.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Was this file included by mistake?  Or is it something that would be useful for all VS Code users?